WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java

```diff
@@ -87,6 +88,7 @@ public class ReaderPostAdapter extends RecyclerView.Adapter<RecyclerView.ViewHol
     private final ImageManager mImageManager;
     private final UiHelpers mUiHelpers;
     private final NetworkUtilsWrapper mNetworkUtilsWrapper;
+    private final LifecycleCoroutineScope mScope;
```

Comment:
Nitpick: Declare field as `CoroutineScope` instead of the more specific `LifecycleCoroutineScope`. I personally think it's a good practice to only be as specific as you need when declaring variables, so if you don't really need specific behavior from a subclass, just using the superclass allows for more flexibility and less coupling.

But this is personal preference, so feel free to ignore this suggestion if you think otherwise.